integration/kubernetes/nginx.bats

```diff
@@ -18,19 +18,22 @@
 load "${BATS_TEST_DIRNAME}/lib.sh"
 
 setup() {
+	service_name="nginx"
+	image_name="$service_name"
 	export KUBECONFIG=/etc/kubernetes/admin.conf
 	master=$(hostname)
 	sudo -E kubectl taint nodes "$master" node-role.kubernetes.io/master:NoSchedule-
+	sudo -E crioctl image pull busybox
```

Comment:
If we are defining `nginx` in a var, we should probably now also do the same for `busybox` to keep it uniform.

Reply:
make sense, fixed.